src/include/daos_security.h

```diff
@@ -180,6 +283,15 @@ daos_ace_free(struct daos_ace *ace);
 ssize_t
 daos_ace_get_size(struct daos_ace *ace);
 
+/**
+ * Print the Access Control Entry to stdout in a human-readable format.
+ *
+ * \param	ace	Access Control Entry to print
+ * \param	tabs	Number of tabs to indent at top level
+ */
+void
+daos_ace_dump(struct daos_ace *ace, uint tabs);
```

Comment:
What do you all think of having the indentation param visible externally? Should I add it to the `daos_acl_dump` method, too?

Reply:
I am fine either way.